hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,17 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ $platform == "windows/amd64" ]]; then
+        os::build::version::get_vars
+        go get -u github.com/josephspurrier/goversioninfo
```

Comment:
Sorry, no. We can install the dependency we need on the CI machines by adding to [this list](https://github.com/openshift/origin-ci-tool/blob/master/oct/ansible/oct/playbooks/prepare/golang.yml#L62-L69). I just meant that we should keep the logic for installing dependencies out of our scripts here -- use `os::util::ensure` to print a nice message when something is missing, but heretofore we have not been in the business of installing stuff on people's machines when they run a script.